Source/WebKit/UIProcess/WebPageProxy.cpp

```diff
@@ -12446,6 +12447,13 @@ void WebPageProxy::shouldAllowDeviceOrientationAndMotionAccess(IPC::Connection&
 
     protectedWebsiteDataStore()->protectedDeviceOrientationAndMotionAccessController()->shouldAllowAccess(*this, *frame, WTFMove(frameInfo), mayPrompt, WTFMove(completionHandler));
 }
+
+bool WebPageProxy::originHasDeviceOrientationAndMotionAccess(const WebCore::SecurityOriginData& origin)
+{
+    // FIXME: implement this.
+    return true;
+}
+
```

Comment:
Will this be implemented in a follow-up patch?

Reply:
Should we file a bug to implement this?

Reply:
Yes, it's in https://github.com/WebKit/WebKit/pull/42211.